dao/mongodb.rya/src/main/java/mvm/rya/mongodb/instance/MongoDetailsAdapter.java

```diff
@@ -167,7 +164,7 @@ public static RyaDetails toRyaDetails(final DBObject mongoObj) throws MalformedR
         return RyaDetails.builder()
             .setRyaInstanceName(basicObj.getString(INSTANCE_KEY))
             .setRyaVersion(basicObj.getString(VERSION_KEY))
-            .setEntityCentricIndexDetails(new EntityCentricIndexDetails(basicObj.getBoolean(ENTITY_DETAILS_KEY)))
+            .setEntityCentricIndexDetails(new EntityCentricIndexDetails(false))//entity indexing is not supported
             .setGeoIndexDetails(new GeoIndexDetails(basicObj.getBoolean(GEO_DETAILS_KEY)))
             .setPCJIndexDetails(getPCJIndexDetails(basicObj))
```

Comment:
PCJs also aren't supported by the Mongo version of this stuff, so it's kind of weird that you've forced the entity centric stuff to always be false, but actually deserialize a DBObject here for the PCJ stuff. I think it makes more sense to handle that nuance outside of this class. Either always make sure these field are disabled before you even serialize them or change the RyaDetails to make these indices optional.